mobile/components/views/Attractions/AddNew/AddNew.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, TextInput, Button, TouchableOpacity } from 'react-native';
+import StarRating from 'react-native-star-rating';
+import OpeningHoursModal from './OpeningHour';
+import mapArrayToJson from '../tools/openingHourMapper';
+
+const AddNewAttraction = () => {
+  const [name, setName] = useState('');
+  const [coordinates, setCoordinates] = useState('');
+  const [description, setDescription] = useState('');
+  const [rating, setRating] = useState(0);
+  const [modalVisible, setModalVisible] = useState(false);
+  const [openingHours, setOpeningHours] = useState(
+    [
+      { day: 'Mon', openingHour: '09:00', closingHour: '17:00' },
+      { day: 'Tue', openingHour: '09:00', closingHour: '17:00' },
+      { day: 'Wed', openingHour: '09:00', closingHour: '17:00' },
+      { day: 'Thu', openingHour: '09:00', closingHour: '17:00' },
+      { day: 'Fri', openingHour: '09:00', closingHour: '17:00' },
+      { day: 'Sat', openingHour: '09:00', closingHour: '17:00' },
+      { day: 'Sun', openingHour: '09:00', closingHour: '17:00' },
+    ]
+  );
```

Comment:
this is double

Reply:
Opening Hours Modal ?

Reply:
Let's leave it for now. Adding attractions is a matter of discussion